samples/qa-tablayout/README.md

```diff
@@ -0,0 +1,9 @@
+## QA sample for testing tablayout

```

Comment:
Let's use `TabLayout` instead of tablayout.

Reply:
It's fixed